lib_xua/src/core/endpoint0/xua_endpoint0.c

```diff
@@ -459,6 +476,7 @@ static unsigned char hidReportDescriptorPtr[] = {
 #endif
 
 
+extern void SetDFUFlag(unsigned x);
```

Comment:
I think this should be added to xua_dfu.h?

Reply:
removed it